src/template/componentsTypescrip.ts

```diff
@@ -0,0 +1,67 @@
+export const felaComponent = (name: string, dependencies: boolean) => `${
+    dependencies
+      ? "import { React,  FunctionComponent } from '../../dependencies';"
+      : "import React, { FunctionComponent } from 'react';"
+  }
+
+type ${name}Props = {
+    styles: object
+}
```

Comment:
I personally prefer 

```ts
interface ${name}Props = {
   styles: object;
}

Reply:
you can also use `FelaWithStylesProps` here 

```ts

export interface ${name}OwnProps {}

type ${name}Props = ${name}OwnProps &  FelaWithStylesProps<${name}OwnProps, {}> 
```

Reply:
Good idea, I changed it 👍